### PR TITLE
Add -d option to script/server command

### PR DESCRIPTION
**Pull request by mallorydxw**
This will allow docker to run in detached mode, so you don't need to use a separate window for running other commands.

### CODE REVIEW

**Comment by mallorydxw** [comment]
Sorry for leaving this PR for ages.

You're using `bash` features here:

```
script/server: 5: [: unexpected operator
```

```
% /bin/sh -c 'if [ "x" == "y" ] ; then echo true; else echo false; fi'                
/bin/sh: 1: [: x: unexpected operator
false
% /bin/bash -c 'if [ "x" == "y" ] ; then echo true; else echo false; fi'
false
```

I believe in `sh` you need to use `test` instead (note that is uses `=` instead of `==` for comparisons):

```
% /bin/sh -c 'if test "x" = "y" ; then echo true; else echo false; fi' 
false
```

**Comment by mallorydxw** [comment]
#45 got closed when renaming `master` to `main`.

**Comment by mallorydxw** [comment]
@RobjS I should probably say your name here because this is a new PR I had to create when renaming the main branch.

File: raw_templates.go [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
...----------------------------------------------------------'\necho 'DEPRECATED'\necho 'This command is deprecated and will be removed in a future version. Please use'\necho 'script/console instead.'\necho '------------------------------------------------------------------------------'\n\nexec script/console\n", "templates/bin/setup.tmpl": "#!/bin/sh\nset -e\n\necho '------------------------------------------------------------------------------'\necho 'DEPRECATED'\necho 'This command is deprecated and will be removed in a future version. Please use'\necho 'script/setup instead.'\necho '------------------------------------------------------------------------------'\n\nexec script/setup\n", "templates/bin/wp.tmpl": "#!/bin/sh\nset -e\n\nFLAGS=\n\n# Add -t flag iff STDIN is a TTY\nif [ -t 0 ]; then\n  FLAGS=-t\nfi\n\nCONTAINER=`docker-compose ps -q wordpress`\n\n# We can't use docker-compose here because docker-compose exec is equivalent\n# to docker exec -ti and docker-compose exec -T is equivalent to\n# docker exec. There is no docker-compose equivalent to docker exec -i.\n#\n# Issue: https://github.com/docker/compose/issues/3352\n\ndocker exec -i ${FLAGS} ${CONTAINER} wp \"${@}\"\n"}
```
Comment (erbridge) [inline raw_templates.go]: What's the change here? The very long line is very hard to parse.

Comment (RobjS) [reply raw_templates.go]: This file is compiled from the files in `templates/` (so the change here is essentially a compiled version of the change in `templates/script/server.tmpl`). We should probably mark it as a binary in `.gitattributes` to make that clear.

Comment (erbridge) [reply raw_templates.go]: I'm not a fan of using `.gitattributes` to hide changes. Making the generator add something to this file that says that it's generated would help, though.

Comment (mallorydxw) [reply raw_templates.go]: This is a compiled file. Its contents are generated from the contents of `templates/` by running `go generate` which in turn runs `go run generate.go`. It generates an AST and prints that to `raw_templates.go`.

Maybe we should `echo /raw_templates.go -diff >> .gitattributes`

Comment (erbridge) [reply raw_templates.go]: Funny that both of you said the same thing :). In sync!

Comment (mallorydxw) [reply raw_templates.go]: https://github.com/dxw/wpc/pull/50

**Comment by RobjS** [comment]
@mallorydxw `if` test fixed to remove the bash format, so this should be ready for another look now.

**Review by mallorydxw — COMMENTED** [review]
It's working for me 👍 
(I can't approve as this PR technically belongs to me)
